projects/cdk/jest.config.ts

```diff
@@ -0,0 +1,11 @@
+import {Config} from 'jest';
+
+import rootConfig from '../../jest.config';
+
+const config: Config = {
+    ...rootConfig,
+    collectCoverageFrom: [`<rootDir>/projects/cdk/**/*.ts`],
```

Comment:
This `jest.config.ts` is different from other `jest.config.ts`.
Is it expected ?

Reply:
fixed